src/components/JobRequestCreation/JobRequestCreationContext.tsx

```diff
@@ -0,0 +1,73 @@
+import {
```

Comment:
This is Context that the execute button and the job/request creation page use. The job/request creation page needs to access the system and command that the job or request refers to. This provides that, but this will likely complicate the eventual re-implementation of the "Pour it again" button on the individual request page, but it should be manageable.